lib/rules/disallow-tabs.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Disallows tabs everywhere.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowTabs": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * if (true) {
+ * \s\sfoo();
+ * }
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * if (true){
+ * \tfoo();
+ * }
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(options) {
+        assert(
+            options === true,
+            this.getOptionName() + ' option requires a true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'disallowTabs';
+    },
+
+    check: function(file, errors) {
+        file.getLines().forEach(function(line, i) {
+            if (line.match(/\t/)) {
+                errors.add('Tab found', i + 1, 0);
```

Comment:
https://github.com/ottumm/node-jscs/commit/e0dad2ebd3c1089410e950febd76d2a7c17ea8af addresses this.